### ISSUE TITLE
[Linux] Interactions with overlay does not work and breaks MTGA

### ISSUE THREAD

**Issue by Porkepix** [Manuel-777/MTG-Arena-Tool#645]
Context informations:
Built from source (current master)
I currently try and move between different window managers or desktop environment. Current bug report is about issues I got with KDE/Plasma, and I guess X.org.
MTGA run through Lutris/Wine.

Overlay correctly appears over the game, but clicks or scrolls doesn't work (can't open settings, can't scroll the overlay)

On top of that, if I don't try to have any interaction with the overlay, keyboard shortcuts does work in MTGA, and if I try to have interaction, it seems to capture keyboard inputs and breaks shortcuts in the game.

**Comment by Manuel-777** [comment]
Please try latest version in master, we believe it solve most (if not all) issues with the overlays;

**Comment by Porkepix** [comment]
> Please try latest version in master, we believe it solve most (if not all) issues with the overlays;

Nope, sadly it's still the same on my current Linux setup: I didn't even tried to interact with the overlay, keyboard input is still stolen from Arena, and clicks not caught by the overlay: I can see through it the clicks effects on Arena's boards.

**Comment by Manuel-777** [comment]
Then this might be specific to KDE, on Gnome it works without any issues. I will try to test on KDE later.

**Comment by Porkepix** [comment]
Okay, so after further testing with different Window managers/Desktop Environment, it doesn't seems to depends it.

I ran test with Gnome (Both X.org and Wayland), Budgie, XFCE (X.org for those I think).
The more detailed result of these tests is as it:

- Setup is a laptop, Full HD (1920x1080), Lutris & Wine, completely up-to-date (ArchLinux, bleeding edge rolling release distro). Game runs as fullscreen, even when game is windowed due to Lutris/Wine running a background desktop.
Single screen, overlay is *over* the game or the virtual desktop.

- When game is on foreground, game picks priority on the mouse/touchpad interactions, which means overlay does not receive them. If I alt-tab and put any other window than the game on foreground, overlay keeps being on-top and now in grab mouse/touchpad interactions.
- The strangest part is about the keyboard: by default shortcuts will work in the game. If I try any mouse interaction with overlay, this interaction won't work but will break keyboard in the game. My guess is that overlay grabs keyboard inputs, but I don't know how to test those.
Here, if I alt-tab out of the game, and alt-tab again in the game, it fixes keyboard inputs… until I try again to click the overlay.

Worth noting, but might deserve another issue: on Linux, checking the "always on top" option or unchecking it changes absolutely nothing: overlay keeps being on-top or everything else.

**Comment by fph** [comment]
I am affected by this bug, too (latest Ubuntu / Gnome / Lutris).
